### DISCUSSION TITLE
Fail to pass Chinese word as url parameter

### DISCUSSION THREAD

**Discussion by Roger-Z** [fastapi/fastapi#8471] [Questions] [answered]
### First Check

- [X] I added a very descriptive title to this issue.
- [X] I used the GitHub search to find a similar issue and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/samuelcolvin/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
import logging

import fastapi
import uvicorn
import socket
from typing import Optional

from logger import Logger
server = fastapi.FastAPI()
log = Logger("service_logs")

@server.get('/param/')
async def param_issue_test(param1:Optional[str] = None):
    logging.info(">> # param1-{}".format(param1))

    return 'success'


if __name__ == '__main__':
    def get_host_ip():
        my_socket = socket.socket(socket.AF_INET, socket.SOCK_DGRAM)
        my_socket.connect(('8.8.8.8', 80))
        current_ip = my_socket.getsockname()[0]
        my_socket.close()
        return current_ip

    current_ip = get_host_ip()
    uvicorn.run(server, host=current_ip, port=5123, debug=False)
```

### Description

curl 'http://xx.xx.xx.xx:5123/param/?param1=欢迎' returns Invalid HTTP request received.

### Operating System

macOS

### FastAPI Version

0.75.2

### Python Version

3.8.5 64-bit

**Comment by yinziyan1206** [comment, marked as answer]
我这边试了是可以的，看下是不是中文转码出现问题了。使用的ssh终端中文url转码或者curl本身url转码有异常

**Reply by zhixiangjoy** [reply]
我也遇到一样的问题，不知道怎么解决。
